pkg/productinfo/google/productinfo_google.go

```diff
@@ -112,6 +112,7 @@ func (g *GceInfoer) Initialize(ctx context.Context) (map[string]map[string]produ
 	log := logger.Extract(ctx)
 	log.Debug("initializing price info")
 	allPrices := make(map[string]map[string]productinfo.Price)
+	unsupportedInstanceType := []string{"n1-ultramem-80", "n1-megamem-96", "g1-small", "n1-ultramem-160", "n1-ultramem-40", "f1-micro"}
```

Comment:
I guess this list is necessary to compare the value of an instance type. Using a map for that might be quicker: instance name in the key, bool (true) in the value.